io.github.hurricangame.hurrican.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general, it's better to maintain this and the desktop files in your repository, so you can update them at the same time you make a release.